src/coreclr/src/vm/threads.h

```diff
@@ -969,6 +969,35 @@ EXTERN_C void STDCALL OnHijackWorker(HijackArgs * pArgs);
 class BaseStackGuard;
 #endif
 
+struct PortableTailCallFrame
+{
+    PortableTailCallFrame* Prev;
+    void* TailCallAwareReturnAddress;
+    void* NextCall;
+};
+
+class TailCallTls
+{
+    friend class MscorlibBinder;
+
+    const PortableTailCallFrame* m_frame;
```

Comment:
The content of this is not really `const`. It may be better to cast-away the constness for the initialization.

Reply:
Good point! Fixed.